cnquery.go

```diff
@@ -42,6 +47,47 @@ func GetVersion() string {
 	return Version
 }
 
+// Release represents a GitHub release
+type Release struct {
+	Name string `json:"name"`
+}
+
+// GetLatestReleaseName fetches the name of the latest release from the specified GitHub repository
+func GetLatestReleaseName() (string, error) {
+	url := "https://api.github.com/repos/mondoohq/cnquery/releases/latest"
+	resp, err := http.Get(url)
+	if err != nil {
+		return "", fmt.Errorf("error fetching latest release: %v", err)
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusOK {
+		return "", fmt.Errorf("received non-OK response status: %s", resp.Status)
+	}
+
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return "", fmt.Errorf("error reading response body: %v", err)
+	}
+
+	var release Release
+	if err := json.Unmarshal(body, &release); err != nil {
+		return "", fmt.Errorf("error unmarshalling response: %v", err)
+	}
+
+	return release.Name, nil
+}
+
+// GetLatestVersion returns the latest version available on Github
+func GetLatestVersion() string {
+	releaseName, err := GetLatestReleaseName()
+	if err != nil {
+		panic(err)
```

Comment:
i dont think we want to panic here, failing to get latest version should not be a cause panic on the status call, we should just push the err through

Reply:
Right, changed it